### DISCUSSION TITLE
Zigbee Fingerbot json format and commands

### DISCUSSION THREAD

**Discussion by afgcons** [arendst/Tasmota#22675] [Q&A]
The (Tuya/)ZigBee Fingerbot sends this
### MQT: tele/zb1/SENSOR = {"ZbReceived":{"0xF687":{"Device":"0xF687","Name":"zs_vherc","EF0006":"00166902000400000064","Endpoint":1,"LinkQuality":63}}}
json message to the ZB-GW03 gateway. And that's all. Can anyone help me how to issue commands and interpret the response?

**Comment by s-hadinger** [comment]
This is Tuya proprietary stuff. The best you can find is doing online search, or maybe checking Zigbee2MQTT
